types/rangy/rangy-classapplier.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for Rangy Class Applier module
+// Project: https://github.com/timdown/rangy
+// Definitions by: Maxime Kjaer <https://github.com/MaximeKjaer>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+///<reference path="index.d.ts"/>
+
+interface RangyStatic {
+    createClassApplier(theClass: string, options?: RangyClassApplierOptions, tagNames?: Array<string> | string): RangyClassApplier;
+}
+
+interface RangyClassApplierOptions {
+    elementTagName?: string;
+    elementProperties?: {[property: string]: any};
+    elementAttributes?: {[attribute: string]: any};
+    ignoreWhiteSpace?: boolean;
+    applyToEditableOnly?: boolean;
+    tagNames?: Array<string> | string;
+    normalize?: boolean;
+    onElementCreate?: (element: Element, classApplier: RangyClassApplier) => void;
+    useExistingElements?: boolean;
+}
+
+interface RangyClassApplier extends RangyClassApplierOptions {
```

Comment:
Yeah that should be tightened up a bit. But `getNativeTextRange` isn't part of the Class Applier module, so I'll leave that for a separate PR.

My intention here was to do a first PR of a manageable size to get a feel for the whole process of submitting to DefinitelyTyped, and to do a bigger PR solving all the issues with Rangy after that (there are some other modules that would need to be typed, and some methods like `rangy.init()` are still undefined here). I'm happy to take care of them, just in the next PR 😊